examples/docker_provider/Gemfile

```diff
@@ -0,0 +1,5 @@
+source 'http://rubygems.org/'
+
+gem 'test-kitchen'
+gem 'kitchen-terraform'
+gem 'kitchen-dokken'
```

Comment:
We could pull `kitchen-dokken` in to the project and create a transport plugin that can handle SSH and Docker. ^_^